prov/gni/src/gnix_vc.c

```diff
@@ -682,10 +682,21 @@ static int __gnix_vc_connect_to_self(struct gnix_vc *vc)
 	vc->peer_irq_mem_hndl = ep->nic->irq_mem_hndl;
 
 	ret = _gnix_vc_schedule(vc);
-	if (ret != FI_SUCCESS)
+	if (ret == FI_SUCCESS) {
+		/*
+		 * give the TX queue a push
+		 */
+		ret = __gnix_vc_push_tx_reqs(vc);
```

Comment:
actually looks like ```__gnix_vc_push_tx_reqs``` is quite verbose for error return codes if compiled with --enable-debug and FI_LOG_LEVEL is set to debug.  I'll remove the warnings from this PR.